src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -1,122 +1,77 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { InjectAppServices } from '../../../services/pure-di';
 import { FormattedMessage } from 'react-intl';
 import { FormattedDateRangeText } from '../../shared/FormattedDateRangeText/FormattedDateRangeText';
 
-class ReportsBox extends React.Component {
-  /**
-   * @param { Object } props - props
-   * @param { import('../../../services/pure-di').AppServices } props.dependencies
-   */
-  constructor({ dependencies: { datahubClient } }) {
-    super();
+/**
+ * @param { Object } props - props
+ * @param { import('../../../services/pure-di').AppServices } props.dependencies
+ */
+const ReportsBox = ({
+  domainName,
+  dateFrom,
+  dateTo,
+  withEmail,
```

Comment:
maybe can use a constant `emailFilter` with the values 'with_email' 'without_email or null